Library/Output Templates/Text - PC.txt

```diff
@@ -10,6 +10,7 @@ Disadvantages:@DISADVANTAGES_LOOP_START @DESCRIPTION_PRIMARY [@POINTS];@DISADVAN
 Quirks:@QUIRKS_LOOP_START @DESCRIPTION_PRIMARY [@POINTS];@QUIRKS_LOOP_END
 Skills:@SKILLS_LOOP_START @DESCRIPTION_PRIMARY @RSL [@POINTS]-@SL;@SKILLS_LOOP_END
 Equipment:@EQUIPMENT_LOOP_START @QTY @DESCRIPTION_PRIMARY [$@COST; @WEIGHT];@EQUIPMENT_LOOP_END
+Spell:@SPELLS_LOOP_START @DESCRIPTION_PRIMARY [Class: @CLASS; Cost: @MANA_CAST; Maintain: @MANA_MAINTAIN; Time: @TIME_CAST ; Duration:@DURATION];@SPELLS_LOOP_END
```

Comment:
Shouldn't this be "Spells:", i.e. plural, like all the rest?

Reply:
Also, unlike all the other things that cost points, you're not listing the points spent in the spell, which seems wrong.

Reply:
done, added RSL as well